src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +32,19 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
```

Comment:
lets swap parameters

Reply:
I dont know why but at my local version I have:
public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {

Reply:
Authentication has to go after @RequestParam, that's what i meant